engine/src/test/java/com/datatorrent/stram/codec/DefaultStatefulStreamCodecTest.java

```diff
@@ -214,4 +214,16 @@ public void testInnerClassSerialization() throws Exception
     }
   }
 
+  static class MyStreamCodec extends DefaultStatefulStreamCodec<String>
```

Comment:
I checked, it can't be private as it will lead to instantiation error. For the name I suggest:
```
static class NoNewInstanceStatefulStreamCodec<T> extends DefaultStatefulStreamCodec<T>
{
}
```